.gitlab-ci.yml

```diff
@@ -601,7 +601,10 @@ build_system-probe-arm64:
     # Artifacts and cache must live within project directory but we run omnibus in a neutral directory.
     # Thus, we move the artifacts at the end in a gitlab-friendly dir.
     # Use --skip-deps since the deps are installed by `before_script`.
-    - inv -e agent.omnibus-build --release-version "$RELEASE_VERSION" --major-version "$AGENT_MAJOR_VERSION" --python-runtimes "$PYTHON_RUNTIMES" --base-dir $OMNIBUS_BASE_DIR ${USE_S3_CACHING} --skip-deps
+    - $S3_CP_CMD $S3_ARTIFACTS_URI/system-probe.${PACKAGE_ARCH} /tmp/system-probe
+    - chmod 755 /tmp/system-probe
+    - $S3_CP_CMD $S3_ARTIFACTS_URI/libbcc-${PACKAGE_ARCH}.tar.xz /tmp/libbcc.tar.xz
+    - SYSTEM_PROBE_BIN=/tmp/system-probe LIBBCC_TARBALL=/tmp/libbcc.tar.xz inv -e agent.omnibus-build --release-version "$RELEASE_VERSION" --major-version "$AGENT_MAJOR_VERSION" --python-runtimes "$PYTHON_RUNTIMES" --base-dir $OMNIBUS_BASE_DIR ${USE_S3_CACHING} --skip-deps
```

Comment:
Small nit: `SYSTEM_PROBE_BIN` and `LIBBCC_TARBALL` are passed as env vars, while everything else is passed as arguments to invoke (which in turn converts to env vars [1]). To be consistent (and also making these parameters more easily discoverable), can you change the env vars to arguments?

[1] https://github.com/DataDog/datadog-agent/blob/master/tasks/agent.py#L404

Reply:
Indeed !
Change implemented.
Thanks for the advice.